suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx

```diff
@@ -96,14 +96,16 @@ export const NetworkSymbolSwitchItem = ({
     };
 
     return (
-        <Card style={applyStyle(wrapperStyle, { isEnabled })}>
-            <View style={applyStyle(iconWrapperStyle)}>
-                <CryptoIcon symbol={networkSymbol} />
-            </View>
-            <HStack justifyContent="space-between" spacing={12} flex={1} alignItems="center">
-                <Text variant="callout">{name}</Text>
-                <Switch onChange={handleEnabledChange} isChecked={isEnabled} />
-            </HStack>
-        </Card>
+        <Pressable onPress={_ => handleEnabledChange(!isEnabled)} accessibilityRole="togglebutton">
```

Comment:
Why not touchable opacity so you have some feedback actually?

Reply:
0b6473418b